platformio.ini

```diff
@@ -59,7 +59,7 @@ upload_protocol = stlink
 [env:fancy-node-usb]
 extends = fancy-node-base
 upload_protocol = custom
-upload_command = stm32loader -p "/dev/ttyUSB0" -e -w -v "$SOURCE"
+upload_command = stm32loader -p "/dev/ttyUSB0" -e -w -v -f G0 "$SOURCE"
```

Comment:
What is `-f`?

Reply:
This sets the microcontroller family. With this set, it displays the size of flash storage (and hides a warning about setting this flag).